src/relentless/simulate/lammps.py

```diff
@@ -393,8 +444,12 @@ def _convert_to_data_file(self, sim):
                         "Atomic atom style is not compatible with topology data."
                     )
                 # store topology data
+                sim[self]["_bonds"] = None
                 if snap.has_bonds():
                     sim[self]["_bonds"] = snap.bonds
+                sim[self]["_angles"] = None
+                if snap.has_angles():
+                    sim[self]["_angles"] = snap.angles
```

Comment:
While testing the inheritance, I discovered that in LAMMPS we ran into an issue where if `"exclude":True` for `rdf` without having that type of connection in the snapshot, then it would raise an error because the key was never defined. This addresses that problem.

Reply:
Makes sense! This is an appropriate fix.